js/src/product-feed/product-statistics/create-campaign-notice.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AddPaidCampaignButton from '.~/components/paid-ads/add-paid-campaign-button';
+import useMCProductStatistics from '.~/hooks/useMCProductStatistics';
+import useAdsCampaigns from '.~/hooks/useAdsCampaigns';
+
+const CreateCampaignNotice = () => {
+	const { hasFinishedResolution: productsResolution, data: products } =
```

Comment:
Renaming the `hasFinishedResolution` prop here is ok, but for readability, let's stick with the `hasFinished....` prefix
```suggestion
	const { hasFinishedResolution: hasMCProductStatsFinishedResolution, data: products } =
```

Reply:
I removed alias, this was the only  `hasFinishedResolution` in that file.